prereise/gather/flexibilitydata/doe/doe_data.py

```diff
@@ -0,0 +1,85 @@
+import os
+import zipfile
+import tarfile
+import urllib.request
+
+'''
+    cleanup_directory(root)
+    
+Recursively cleanup a folder by deleting meaningless or empty files
+'''
+def cleanup_directory(root):
+
+    all_files = os.listdir(root)
+    for i in all_files:
+        fp =  root+'/'+i
+        if i[0] == '.' or (i[-3:]=='csv' and os.path.getsize(fp) < 0.01):
+            os.remove(fp)
+
+    all_folders = os.listdir(root)
+    for i in all_folders:
+
+        if not i[-3:] == 'csv':
+            cleanup_directory(root+i)
```

Comment:
> Here are my comments on `doe_data.py` and `geo_data.py`. Also, please don't forget to run `black`, which should probably cause the linting tests to pass.

Hello Lane, thank you for reading through the code and providing your comments. This version was meant to present a rough implementation of the entire workflow discussed in the meetings. I did not carefully check the code because it will definitely be reworked in the future. I have addressed your comments in the new coming commit.

Thanks!

Reply:
One thing that is often helpful in situations like this is to make a draft PR instead of a full one, and to specify which parts are most ready for review and which are the least certain. E.g. sometimes the low-level functions are in good shape and the higher-level strategy for how to use them is still TBD, or vice versa. This helps the reviewers know which things to focus on, and they can provide better feedback on those specific aspects rather than devoting their time to getting through the whole thing.